InkyPhat/app/src/main/java/com/novoda/inkyphat/ShapeDrawer.java

```diff
@@ -0,0 +1,422 @@
+package com.novoda.inkyphat;
+
+import static java.lang.Math.cos;
+import static java.lang.Math.sin;
+
+public class ShapeDrawer {
```

Comment:
an alternative approach would be leveraging the drawing functions that already exist in the Android SDK's [`Canvas`](https://developer.android.com/reference/android/graphics/Canvas.html) backed by a `Bitmap` of the size of the inkyphat display and then you'd only need 1 method to draw a bitmap onto the inkyphat that way you'd avoid a lot of boilerplate which means potential of less bugs 💃